packages/core/src/index.ts

```diff
@@ -60,6 +61,10 @@ export {
   generateId
 } from './utils/public';
 
+export {
+  parsePluginName
+} from './utils/utils';
```

Comment:
Hmm, `utils` is duplicated with the folder, suggestion: `internals` or `common`.